include/envoy/filter/http/filter_config_provider.h

```diff
@@ -32,11 +32,14 @@ class FilterConfigProviderManager {
    * @param filter_config_name the filter config resource name.
    * @param factory_context is the context to use for the filter config provider.
    * @param stat_prefix supplies the stat_prefix to use for the provider stats.
+   * @param last_filter_in_current_config indicate the filter position at filter chain
+   * @param filter_chain_type is the filter chain type
    */
   virtual FilterConfigProviderPtr createDynamicFilterConfigProvider(
       const envoy::config::core::v3::ExtensionConfigSource& config_source,
       const std::string& filter_config_name, Server::Configuration::FactoryContext& factory_context,
-      const std::string& stat_prefix) PURE;
+      const std::string& stat_prefix, bool last_filter_in_current_config,
```

Comment:
maybe just `last_filter`? or `last_filter_in_filter_chain`? not sure what "current" is referring to

Reply:
`last_filter_in_filter_chain` better.